src/app/lib/utilities/serialiseForScript/index.test.js

```diff
@@ -0,0 +1,37 @@
+import serialiseForScript from '.';
+
+describe('serialiseForScript', () => {
+  describe('objects', () => {
+    it('an empty object', () => {
+      expect(serialiseForScript({})).toBe('{}');
+    });
+
+    it('an example object with no special characters', () => {
+      expect(serialiseForScript({ a: 1, b: 'foo', c: '' })).toBe(
+        '{"a":1,"b":"foo","c":""}',
+      );
+    });
+
+    it('an object with a less-than symbol in a property value', () => {
+      expect(serialiseForScript({ text: 'me < you' })).toBe(
+        '{"text":"me \\u003c you"}',
+      );
+    });
+
+    it('an object with a less-than symbol in a property name', () => {
+      expect(serialiseForScript({ '<blink>': 'hello!' })).toBe(
+        '{"\\u003cblink>":"hello!"}',
+      );
+    });
+  });
+
+  describe('unexpected input', () => {
+    it('null', () => {
+      expect(serialiseForScript(undefined)).toBeUndefined();
```

Comment:
Is this meant to be?:
```js
expect(serialiseForScript(null)).toBeNull();
```

Reply:
Oops yes, copy-paste error. Ta.

Reply:
Actually, the result is `undefined` due to the short-circuit OR. I don't think it makes much difference in practice though.